src/NServiceBus.AcceptanceTests/CriticalError/When_raising_critical_error.cs

```diff
@@ -65,21 +96,10 @@ public CriticalHandler(CriticalError criticalError, TestContext testContext)
 
                 public Task Handle(Message request, IMessageHandlerContext context)
                 {
-                    try
-                    {
-                        if (testContext.ContextId == request.ContextId)
-                        {
-                            criticalError.Raise("a ciritical error", new SimulatedException());
-                        }
-                    }
-                    catch (Exception exception)
-                    {
-                        testContext.RaiseCriticalErrorException = exception;
-                    }
-
                     if (testContext.ContextId == request.ContextId)
                     {
-                        testContext.CriticalErrorRaised = true;
+                        criticalError.Raise("a critical error", new SimulatedException());
+                        testContext.CriticalErrorsRaised++;
```

Comment:
Interesting, so when critical error is raised, because of async/await, there's still some work that can take place after...